pom.xml

```diff
@@ -527,6 +526,7 @@
         <module>examples/spark2</module>
         <module>datamap/lucene</module>
         <module>datamap/bloom</module>
+        <module>datamap/example</module>
```

Comment:
I think it is better not to add this, since it will make the assembling bigger

Reply:
Excluding this will cause the datamap example module outdated and has potential unfixed bugs later, which is the previous status of this module.

Maybe we can execlude this from the assembly jar